app/components/total-monthly-estimate/template.hbs

```diff
@@ -0,0 +1 @@
+${{totalDollars}}
```

Comment:
@gib and I worked on a pretty formatter for currency in a branch on his machine a little while back (with some integration tests), maybe we could bring that in and have `$3,456.78` instead of `$3456.78`?